server/lib/config/pbench-server-default.cfg

```diff
@@ -126,12 +126,32 @@ lowerbound = 820
 # uri = driver://user:password@hostname/dbname
 wait_timeout = 120
 
+[authentication]
+# Secret key for use by the Pbench Server API sub-system for security related
+# tasks.
+secret-key = my_precious
+
+# User authentication section to use when authorizing the user with an OIDC
+# identity provider.
+
+# URL of the OIDC auth server
+#server_url = http://keycloak-broker.example.com:8080
```

Comment:
Lines 134-136 appear to refer to the `section`, and so they should appear either before line 129 or immediately after it.

Reply:
Fixed.